libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthenticationFactory.cs

```diff
@@ -12,6 +12,15 @@ namespace Microsoft.Bot.Connector.Authentication
     /// </summary>
     public static class BotFrameworkAuthenticationFactory
     {
+        /// <summary>
+        /// Creates the a <see cref="BotFrameworkAuthentication" /> instance for anonymous testing scenarios.
+        /// </summary>
+        /// <returns>A new <see cref="BotFrameworkAuthentication" /> instance.</returns>
+        public static BotFrameworkAuthentication Create()
+        {
+            return Create(null, false, null, null, null, null, null, null, null, new PasswordServiceClientCredentialFactory(), new AuthenticationConfiguration(), null, null);
```

Comment:
That's a scary amount of nulls. For sanity of maintainers, it might be a decent option to make one paramter per line, and add the names, ie. (pseudocode)

```csharp
return Create(
   foo: null,
   bar: null,
   // ...etc
```